packages/puppeteer-extra-plugin-recaptcha/readme.md

```diff
@@ -31,10 +31,11 @@ const puppeteer = require('puppeteer-extra')
 
 // add recaptcha plugin and provide it your 2captcha token
 // 2captcha is the builtin solution provider but others work as well.
+// Funds will need to be added to your 2captcha account in order for the 2captcha provider to work
 const RecaptchaPlugin = require('puppeteer-extra-plugin-recaptcha')
 puppeteer.use(
   RecaptchaPlugin({
-    provider: { id: '2captcha', token: 'XXXXXXX' },
+    provider: { id: '2captcha', apiKey: 'XXXXXXX' },
```

Comment:
We need to change this back to `token` as we'd otherwise need to update the code as well.

Reply:
Good call! My bad for not thinking of that.